data/themes/default/style.css

```diff
@@ -3,7 +3,7 @@
  ********************/
 
 /* most foreground text items */
-QLabel, QTreeWidget, QListWidget, QGroupBox, QMenuBar {
+QLabel, QTreeWidget, QListWidget, QGroupBox, QMenuBar, QCheckBox {
 	color: #d1d8e4;
```

Comment:
We also maintain the **classic** theme. The same code is present there. As I understand it, this regards only the text with the checkboxes in **Settings**.

Reply:
@zonkmachine, with the classic theme the change has no effect. It always looks as follows:

![2510-SetupDialogClassic](https://github.com/LMMS/lmms/assets/9293269/3e0f2964-2ff5-4bd3-9ea3-e02c5b5ad3d1)